tests/integration/test_charm.py

```diff
@@ -204,95 +210,89 @@ async def test_add_relational_db_with_relation_expect_active(self, ops_test: Ops
     @pytest.mark.abort_on_fail
     async def test_can_connect_exporter_and_get_metrics(self, ops_test: OpsTest):
         config = await ops_test.model.applications[CHARM_NAME].get_config()
-        url = f"http://localhost:{config['mlflow_prometheus_exporter_nodeport']['value']}/metrics"
+        exporter_port = config["mlflow_prometheus_exporter_port"]["value"]
+        mlflow_subprocess = subprocess.Popen(
+            [
+                "kubectl",
+                "-n",
+                "kubeflow",
```

Comment:
@misohu Looks like integration tests are failing. Is it because of the namespace we specify here?